src/fsharp/FSharp.Core/map.fsi

```diff
@@ -86,6 +92,14 @@ namespace Microsoft.FSharp.Collections
         [<CompiledName("Add")>]
         val add: key:'Key -> value:'T -> table:Map<'Key,'T> -> Map<'Key,'T>
 
+        /// <summary>Returns a new map with the value stored under key changed according to f.</summary>
+        /// <param name="key">The input key.</param>
+        /// <param name="f">The change function.</param>
+        /// <param name="table">The input map.</param>
+        /// <returns>The resulting map.</returns>
+        [<CompiledName("Change")>]
+        val change: key:'Key -> f:('T option -> 'T option) -> table:Map<'Key,'T> -> Map<'Key,'T>
```

Comment:
I think the attribute needs to also be in the signature

Reply:
It does.